build-library/action.yml

```
@@ -1,34 +1,49 @@
-name: "Build a Python library."
-description: "Build a Python library by generating wheels and sdist artifacts."
+name: >
+  Build library action
+
+description: >
+  Verifies if a Python library builds properly. As a result of a successfull
+  building process, wheel and source distribution artifacts are generated. This
+  action is expected to be used as an independent job.
 
 inputs:
+
+  # Required inputs
+
   library-name:
-    description: 'Name of the Python library.'
+    description: >
+      Name of the Python library. This is the name used when uploading the wheel
+      and source distribution as artifacts. The name should be the same one used
+      in the PyPI index.
     required: true
     type: string
+
+  # Optional inputs
+
   python-version:
-    description: 'Desired Python version for build.'
+    description: >
+      Python version used for installing and running ``pre-commit``.
     default: '3.10'
     required: false
     type: string
+
   use-python-cache:
-    description: 'Whether to use Python cache or not. Requires a pyproject.toml file.'
+    description: >
+      If ``true``, uses Python cache to install previously downloaded
+      libraries. When set to ``false``, all the required libraries are
```

```suggestion
      libraries. If ``false``, all the required libraries are
```
I would like to see a general description before indicating what "true" does versus what "false" does.  Can the description start with "Whether to use the Python cache for installing previously downloaded libraries. If ``true``, previously downloaded libraries are installed form the Python cache. If ``false``, ...."?

---

I agree about this. I will apply this suggestion from now on. Thanks for the tip, Kathy.